### DISCUSSION TITLE
Please improve list of available glyphs

### DISCUSSION THREAD

**Discussion by cachocarne** [chromiecraft/chromiecraft#5739] [Suggestions]
As of now, locked up to TBC / 70 bracket, there is a list of 263 available Inscription recipes, from which 69 of them are "Parts" (inks), vellums, scrolls and some tomes.
That means there are 194 glyphs in game. 20 glyphs of those are for Death Knight, which are not in game yet. So only 174 glyphs, and many of them are not used because the skills are not existing in the game.
Apart from that, at least 6 of the ones left are made with Ink of the Sea, which is a component we cannot create yet because it's made from herbs gathered in Northrend:

· Glyph of Typhoon
· Glyph of Blast Wave
· Glyph of Thunderstorm
· Glyph of Souls
· Glyph of Command
· Glyph of Shadowfiend

This cuts the whole list even shorter.

Honestly, I think having other glyphs available from vendors is a mistake and a damage to the current state, making it simply easier for those with gold, for some reason, who have access to glyphs others can't.

At the same time, there is a BIG list of glyphs still not available to discover or learn because they were added on WotLK or are obtained through Book of Glyph Mastery. However, the glyphs on the following list are not created with WotLK herbs and also are not for DK, which means they could be added to Scribes through Minor Inscription Research or via Trainer.

![ 2023-06-10 a las 10 44 30](https://github.com/chromiecraft/chromiecraft/assets/77953030/dc0b2f83-ce5c-49ce-b4fc-bf44cf08c76d)
![ 2023-06-10 a las 10 44 34](https://github.com/chromiecraft/chromiecraft/assets/77953030/62fab4e0-a5c2-44ce-8379-427a2df6676e)

I don't say there should be all these available already, but maybe at least SOME good glyphs in exchange for the ones we can't even craft now. Remember, there are 20 for DK and 6 more that we cannot craft. 26 useless recipes.

Please consider reworking the available glyphs to give access to useful ones.
Alternatively, place a vendor who can teach us Scribes some without skill restriction 425, so we can also make some money, instead of offering the final product to wealthy players

Thanks.

**Comment by cachocarne** [comment]
Any feedback on this? Suggestions?

**Comment by balleny** [comment]
there are also quite some glyphs that 
a) require a skill level we cannot reach before wotlk
b) are learned through the world random drop in wotlk zones, but require classic / tbc materials. I think you didnt consider these yet

making these wotlk glyphs available now, would mean that you would also have to make the materials available now by some means, and unlearn + remove them once wotlk bracket start.

Instead, we have them available via gold and honor which are both a gold-sink for longtime players or a pvp incentive for most others and hence already strike a good balance

**Reply by cachocarne** [reply]
The only skill not learnt through trainer or Book of Glyph Mastery is the Offhand that transforms you in a wolf (Rituals of the New Moon), drop from specific Silverbrook npcs in Grizzly Hills.
The list of glyphs I posted do NOT require Northrend herbs and are NOT for DK, but didn't went on to check every level for it either.

**Reply by balleny** [reply]
practically, if you enable access to the recipes now, you will either have to remove them (the glyphs and the recipes) from everyone, or you are devaluing the books of glyph mastery

**Reply by cachocarne** [reply]
That's why I propose a CHANGE of glyphs, not that we get glyphs recipes now that we don't need to learn later. There are 26 recipes that they could take away from us now and that we still will need to get later.
